.github/workflows/backend_associated_test_file_check.yml

```diff
@@ -0,0 +1,44 @@
+name: Backend associated test file check
+on:
+  push:
+    branches:
+      - develop
+      - release-*
+  pull_request:
+    branches:
+      - develop
+      - release-*
+
+jobs:
+  run_tests:
```

Comment:
Can you please add a name to it below? Something like [this](https://github.com/oppia/oppia/blob/develop/.github/workflows/backend_tests.yml#L14)

Reply:
Done!